src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -214,3 +214,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('subnet', help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
         c.argument('vnet_name', help='Name of a virtual network.')
         c.argument('ip_address', help='IPv4 address or CIDR range.')
+
+    with self.argument_context('acr check-health') as c:
+        c.argument('registry_name', options_list=['-n', '--name'], help='Name of target registry.', required=False)
+        c.argument('skip_confirmation', options_list=['-y'], help='Skips confirmation, allowing to automatically pull the sample image', action='store_true', required=False)
+        c.argument('ignore_errors', options_list=['--ignore'], help='Ignore errors, displaying them only at the final', action='store_true', required=False)
```

Comment:
I have a minor pick on the param `--ignore` (i.e., ignore what?), the original `--ignore-errors` might be better, unless there is an established convention such as `--yes` that everyone knows.

Reply:
Agreed, moved to `--ignore-errors`.